src/components/settings/index.jsx

```diff
@@ -30,12 +30,10 @@ const SettingsPage = () => {
       <Switch>
         <Redirect
           exact
-          key="settings"
```

Comment:
don't quite get what `key` was doing in the first place ❓

Reply:
Here, it wasn't doing anything haha. `key` are generally used when rendering a node as part of iterating over an array, but that's not the case here.

Another fun fact about `key`: if `key` changes, the component it's on will be forced to re-render!